persistence/mongodb-record/src/main/scala/net/liftweb/mongodb/record/field/MongoListField.scala

```diff
@@ -159,39 +155,10 @@ class MongoListField[OwnerType <: BsonRecord[OwnerType], ListType: Manifest](rec
 
   // set this field's value using a DBObject returned from Mongo.
   def setFromDBObject(dbo: DBObject): Box[MyType] =
-    setBox(Full(dbo.asInstanceOf[BasicDBList].toList.asInstanceOf[MyType]))
+    setBox(Full(dbo.asInstanceOf[BasicDBList].asScala.toList.asInstanceOf[MyType]))
 
   def setFromDocumentList(list: java.util.List[Document]): Box[MyType] = {
     throw new RuntimeException("Warning, setting Document as field with no conversion, probably not something you want to do")
   }
 
-}
-
-/*
-* List of JsonObject case classes
-*/
-class MongoJsonObjectListField[OwnerType <: BsonRecord[OwnerType], JObjectType <: JsonObject[JObjectType]]
```

Comment:
The class isn't really removed, just moved to a more consistent location. I.e. fields for a single instances of a given type and the fields for the list of instances of that same type are generally in the same file. So, I just moved this to a file that contains `JsonObjectField`.